source/components/spinner/spinner.html

```diff
@@ -1,6 +1,6 @@
 <rl-generic-container selector="spinner.ngDisabled">
 	<template default>
-		<div ng-class="{ 'error': spinner.ngModel.$invalid }">
+		<div class="field" ng-class="{ 'error': spinner.ngModel.$invalid }">
```

Comment:
Do you also need to set this on the disabled spinner below? I suppose validation styling in particular isn't very relevant if the spinner is disabled